docs/man/man1/kubectl-stop.1

```diff
@@ -30,6 +30,10 @@ If the resource is resizable it will be resized to 0 before deletion.
     Filename, directory, or URL to file of resource(s) to be stopped
 
 .PP
+\fB\-\-grace\-period\fP=\-1
```

Comment:
Nothing - just ignore the value, but is there any better default value?

What do you mean by "we can't change... "?